website/server/controllers/api-v3/challenges.js

```diff
@@ -204,11 +321,20 @@ api.leaveChallenge = {
 };
 
 /**
- * @api {get} /api/v3/challenges/user Get challenges for a user
+ * @api {get} /api/v3/challenges/user Get challenges for a user.
  * @apiName GetUserChallenges
  * @apiGroup Challenge
+ * @apiDescription Get challenges that the user is a member, public challenges and the ones from the user's groups.
  *
- * @apiSuccess {Array} data An array of challenges sorted with official challenges first, followed by the challenges in order from newest to oldest
+ * @apiSuccess {Object[]} challenges An array of challenges sorted with official challenges first, followed by the challenges in order from newest to oldest
+ * @apiSuccess {Object} challenges.challenge An object representing the details
```

Comment:
I don't think this is necessary. (especially since this makes it look like the success looks like:

```
[
  {
    challenge: { /* challenge data */ },
  }, {
    challenge: { /* challenge data */ },
  }
]
```

When it's more like:

```js
[
  {
    /* challenge data */
  }, {
    /* challenge data */
  }
]
```